src/xcode/ENA/ENASecurity/Sources/ENASecurity/ECKeyPairGeneration.swift

```diff
@@ -0,0 +1,79 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+public struct ECKeyPairGeneration {
+    
+    // MARK: - Public
+
+    public func generatePrivateKey(with name: String? = nil) -> (SecKey?, String?) {
```

Comment:
I think we should have a common API Style. In the other cases we are using Result type with explicit errors.